src/applications/rated-disabilities/tests/e2e/rated-disabilities.cypress.spec.js

```diff
@@ -1,64 +1,110 @@
-import featureToggleDisabled from '../fixtures/feature-toggle-disabled.json';
```

Comment:
Github isn't picking up on this, but I renamed the `rated-disabilities-lighthouse` file to just be `rated-disabilities`